content/blog/2018-03-15-update-on-async-rendering.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Update on Async Rendering
+author: [bvaughn]
+---
+
+For the past few months, the React team has been experimenting with [asynchronous rendering](/blog/2018/03/01/sneak-peek-beyond-react-16.html), and we are very excited about the new features it enables.
```

Comment:
Let's make this a bit more assertive. "Experimenting" seems like we're not sure what we're doing.

Reply:
👍